src/types.ts

```diff
@@ -37,14 +37,14 @@ export type IOState = {
   records: Records; // The list of all ArNS names and their associated data
   gateways: Gateways; // each gateway uses its public arweave wallet address to identify it in the gateway registry
   fees: Fees; // starting list of all fees for purchasing ArNS names
-  settings: ContractSettings; // protocol settings and parameters
   reserved: ReservedNames; // list of all reserved names that are not allowed to be purchased at this time
   auctions: Auctions;
   lastTickedHeight: number; // periodicity management
   demandFactoring: DemandFactoringData;
   observations: Observations;
   distributions: EpochDistributionData;
   vaults: RegistryVaults;
+  settings?: unknown; // satisfying integration tests type issues
```

Comment:
probably don't need this in the app code. going to try scoping this type to where I'm seeing errors in integration